bitnami/apache/templates/deployment.yaml

```diff
@@ -63,7 +63,8 @@ spec:
             - -ec
             - |
               [[ -f "/opt/bitnami/scripts/git/entrypoint.sh" ]] && source "/opt/bitnami/scripts/git/entrypoint.sh"
-              git clone {{ .Values.cloneHtdocsFromGit.repository }} --branch {{ .Values.cloneHtdocsFromGit.branch }} /app
+              git clone {{ .Values.cloneHtdocsFromGit.repository }} --branch {{ .Values.cloneHtdocsFromGit.branch }} /tmp/repo
+              [[ "$?" -eq 0 ]] && rm -rf /app/* && mv /tmp/repo/* /app/
```

Comment:
This doesn't remove `.*` file/directories.
How about this?

Reply:
Hi @cuttingedge1109 ,
AFAIK the `/*` will remove either files or directories

Reply:
@miguelaeh well ofc, it will remove dirs and files but not dirs with the name `.*`, typically `.git` dir is not  deleted with that command.
So i suggest this one.
```
rm -rf ..?* .[!.]* *
```